src/remote.go

```diff
@@ -66,7 +66,8 @@ const (
 var (
 	ErrPathNotExists                  = errors.New("remote path doesn't exist")
 	ErrNetLookup                      = errors.New("net lookup failed")
-	ErrClashesDetected                = fmt.Errorf("clashes detected. use `%s` to override this behavior", CLIOptionIgnoreNameClashes)
+	ErrClashesDetected                = fmt.Errorf("Clashes detected. Use `%s` to override this behavior or `%s` to try fixing this",
```

Comment:
Could we keep the first letter of the error as lowercase? 
It is a good practice to use lowercased beginning letters in error messages, this helps in the case where you compose multiple errors.